cmd/monaco/main.go

```diff
@@ -159,6 +159,7 @@ Examples:
 			ctx.String("specific-environment"),
 			ctx.String("project"),
 			ctx.Bool("dry-run"),
+			ctx.Bool("continue-on-error"),
```

Comment:
I just noticed the flag definition is missing in the flag array above, or am i wrong?

Reply:
Actually intention was to support `continue-on-error` only with new_cli and this param is needed here (will anyways be false) or am I missing something?

Reply:
In that case we should explicitly write `false` in there if you ask me. 
Otherwise it looks like an error to me.